.github/ISSUE_TEMPLATE/onboarding-epic.md

```diff
@@ -0,0 +1,40 @@
+---
+name: Onboarding Epic Template
+about: For filing an onboarding epic on VSP.
+title: ''
+labels: 'product-support', 'onboarding'
+assignees: ''
+
+---
+
+## User Story
+As a new VA.gov team member, I need to complete onboarding activities so I can successfully work on my project.
+
+## Acceptance Criteria
+- [ ] All of the team members have completed the onboarding checklist.
+
+## Definition of Done
+- [ ] General onboarding meeting (full team)
+- [ ] Full team: Review general onboarding documentation
+	- [ ] Review the [Code of Conduct](https://github.com/department-of-veterans-affairs/va.gov-vfs-teams/blob/master/code-of-conduct.md)
+	- [ ] Review the [Platform Principles](https://github.com/department-of-veterans-affairs/va.gov-vfs-teams/blob/master/Norms/platform-principles.md)
+ 	- [ ] Review the [Norms for Communication](https://github.com/department-of-veterans-affairs/va.gov-vfs-teams/blob/master/Norms/norms-communication.md)
+	- [ ] Review the [Norms for using GitHub](https://github.com/department-of-veterans-affairs/va.gov-vfs-teams/tree/master/Norms/Github)
+	- [ ] [Request access to tools](https://github.com/department-of-veterans-affairs/va.gov-vfs-teams/blob/master/Onboarding/request-access-to-tools.md)
+	- [ ] Review the [Norms for using Tools](https://github.com/department-of-veterans-affairs/va.gov-vfs-teams/blob/master/Norms/norms-tools.md)
+- [ ] Designers: Review [Design & Research onboarding documentation](https://design.va.gov/documentation/designers)
+- [ ] Engineers: [set up local dev environment](https://github.com/department-of-veterans-affairs/va.gov-vfs-teams/blob/master/DeveloperDocs/getting-started.md)
+  - [ ] [Create SSH keys](https://github.com/department-of-veterans-affairs/vets.gov-team/blob/master/Practice%20Areas/Engineering/Internal%20Tools.md#create-ssh-public-key)
```

Comment:
I believe there is now an issue template for taking care of a lot of this, with the idea that the user fills out the issue and the Operations team takes it from there. See https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?labels=external-request%2C+operations&template=Environment-Access-Request-Template.md&title=Access+for+%5Bindividual%5D

cc @ricetj @wyattwalter

Reply:
Looks like the template is referenced in one of the docs linked above. Please disregard!